File: hermes/src/main/java/org/eclipse/paho/android/service/contract/ServiceStartActionListener.java
package org.eclipse.paho.android.service.contract;

/**
 * 操作回调
 * @author zhouL
 * @date 2023/9/27
 */
public interface ServiceStartActionListener {

    void onSuccess();

    void onFailure(String errorMsg, Throwable t);
}

File: hermes/src/main/java/com/lodz/android/hermes/mqtt/base/contract/ServiceStartActionListener.kt
package com.lodz.android.hermes.mqtt.base.contract

/**
 * 后台服务启动操作回调
 * @author zhouL
 * @date 2023/10/12
 */
interface ServiceStartActionListener {

    /** 后台服务启动成功 */
    fun onSuccess()

    /** 后台服务启动失败，提示语[errorMsg]，异常[t] */
    fun onFailure(errorMsg: String, t: Throwable)

}